pkg/testutil/restmapper.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package testutil
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+func NewFakeRESTMapper(gvks ...schema.GroupVersionKind) meta.RESTMapper {
```

Comment:
Not sure if it's helpful or something we want to move to, but there is functionality for a testing RESTMapper here:

`/staging/src/k8s.io/apimachinery/pkg/api/meta/testrestmapper/test_restmapper.go:func TestOnlyStaticRESTMapper(scheme *runtime.Scheme, versionPatterns ...schema.GroupVersion) meta.RESTMapper {
`